src/Groups/GAPGroups.jl

```diff
@@ -80,32 +81,69 @@ function parent(x::GAPGroupElem)
   return x.parent
 end
 
-function Base.isfinite(G::PermGroup)
-  return true
-end
+"""
+    isfinite(G::GAPGroup) -> Bool
 
-function Base.isfinite(G::PcGroup)
-  return true
-end
+Return `true` if `G` is finite, and `false` otherwise.
 
-function Base.isfinite(G::GAPGroup)
-  return GAP.Globals.IsFinite(G.X)
-end
+# Examples
+```jldoctest
+julia> isfinite(symmetric_group(5))
+true
+
+julia> isfinite(free_group(2))
+false
+
+```
+"""
+Base.isfinite(G::GAPGroup) = GAP.Globals.IsFinite(G.X)
+
+Base.isfinite(G::PermGroup) = true
+
+Base.isfinite(G::PcGroup) = true
+
+"""
+    isfinite_order(g::GAPGroupElem) -> Bool
+
+Return `true` if `g` has finite order, and `false` otherwise.
+
+# Examples
+```jldoctest
+julia> isfinite_order(gen(symmetric_group(5), 1))
+true
+
+julia> isfinite_order(gen(free_group(2), 1))
+false
+
+```
+"""
+isfinite_order(x::GAPGroupElem) = GAP.Globals.IsInt(GAP.Globals.Order(x.X))
 
 """
     degree(G::PermGroup)
 
-Return the degree as permutation group, that is the integer `n` such that `G < Sym(n)`.
+Return the degree of `G` as permutation group, that is,
+the integer `n` such that `parent(G)` is `symmetric_group(n)`.
 
 !!! warning "Note"
-    This is *not* the smallest `k` such that `G` embeds in `Sym(k)`.
+    This is in general *not* the smallest `k` such that `G` embeds in
+    `symmetric_group(k)`.
 """
-function degree(x::PermGroup)
-   return x.deg
-end
+degree(x::PermGroup) = x.deg
 
-order(x::Union{GAPGroupElem, GAPGroup}) = order(fmpz, x)
+"""
+    order([::Type{T}, ]x::Union{GAPGroupElem, GAPGroup}) where T
```

Comment:
Orthogonal to all this: should we restrict `T` to be `Union{Integer, fmpz}` ? And: do we already have a type alias for this probably very common union?